types/luxon/index.d.ts

```diff
@@ -329,21 +334,21 @@ declare module 'luxon' {
                     seperator?: string;
                 }
             ): string;
-            toISO(options?: Object): string;
+            toISO(options?: ISOTimeOptions): string;
             toString(): string;
             union(other: Interval): Interval;
         }
 
-        type Settings = {
-            defaultLocale: string;
-            defaultNumberingSystem: string;
-            defaultOutputCalendar: string;
-            defaultZone: Zone;
-            defaultZoneName: string;
-            now: Function;
-            throwOnInvalid: boolean;
-            resetCache(): void;
-        };
+        class Settings {
```

Comment:
WHy did you make this a class, and add a rule in the `tslint.config`?

Reply:
Lint rules saids `Every member of this class is static. Use namespaces or plain objects instead.`
For test, it should be namespace, not the type or interface.
But I thought the luxon code is written in class, so class seems to be right. Even there is rare possibility for Settings or Info class will be changed for having instance methods.

How do you think? `namespace` is more clearer than class with static?

Reply:
yeah, namespace is probably better, if/when an instance method is added we can make additional changes then.